src/_pytest/config/__init__.py

```diff
@@ -915,7 +915,7 @@ def parse(self, args, addopts=True):
         assert not hasattr(
             self, "args"
         ), "can only parse cmdline args at most once per Config object"
-        self._origargs = args
+        assert self.invocation_params.args == args
```

Comment:
Just for testing / ensuring - can be removed, and/or improved (msg).

Reply:
@RonnyPfannschmidt / @nicoddemus 
What about this?

Reply:
hmm, that one seems ok to me

Reply:
Agreed, nothing fishy at first glance.

Reply:
I think it should have either be removed (I've added it for a full test run on CI), or improved (i.e. a msg).
But ok then for now.